features/fixtures/sidekiq/app/Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'bugsnag'
+gem 'bugsnag', path: '/bugsnag'
 gem 'sidekiq', ENV['SIDEKIQ_VERSION']
+gem 'timers', '~> 4.1.1'
```

Comment:
Why have these gems been added?

Reply:
A loading/bundler issue with installing the Gems was occuring with this configuration.  This fixes that, ensuring we can test against the Ruby 2.0/Sidekiq 3 combination